src/theme/themeProvider.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import {ThemeContext} from './themeContext';
+
+export const ThemeProvider = props => (
+  <ThemeContext.Provider value={props.value}>
+    {props.children}
```

Comment:
eslint don't underline this line?
Because props can be any, maybe we should define ThemeProvider as React.FC<... some props ...> and than youll see props will have children ...